clients/venice-client/src/main/java/com/linkedin/venice/fastclient/ClientConfig.java

```diff
@@ -206,6 +217,12 @@ private ClientConfig(
         && this.storeMetadataFetchMode != StoreMetadataFetchMode.SERVER_BASED_METADATA) {
       throw new VeniceClientException("Helix assisted routing is only available with server based metadata enabled");
     }
+    this.batchGetDefaultsToStreamingBatchGet = batchGetDefaultsToStreamingBatchGet;
+    if (this.batchGetDefaultsToStreamingBatchGet) {
+      LOGGER.info("BatchGet will use streamingBatchGet");
```

Comment:
NIT: let's avoid camel case or code variables in logs if possible, can just replace with `Batch get will use streaming batch get implementation`.